R/wrappers.R

```diff
@@ -92,14 +91,8 @@ createWorkbook <- function(creator = ifelse(.Platform$OS.type == "windows", Sys.
 #' saveWorkbook(wb, file = "saveWorkbookExample.xlsx", overwrite = TRUE)
 #' }
 saveWorkbook <- function(wb, file, overwrite = FALSE, returnValue = FALSE) {
-  od <- getOption("OutDec")
-  options("OutDec" = ".")
-  on.exit(expr = options("OutDec" = od), add = TRUE)
-
-  ## increase scipen to avoid writing in scientific
-  sci_pen <- getOption("scipen")
-  options("scipen" = 10000)
```

Comment:
after the change this defaults to scipen 200, is that enough? I'm unaware why this specific value was chosen, but we should make sure that we're not regressing in the xlsx output.

Reply:
Some arbitrarily large number is assigned to try to preserve digits during conversions.  I think `200` is used in mostly places but anything large enough should probably be fine.

Reply:
Thanks, perfectly fine for me. Though I'd appreciate @ycphs reviewing this PR too.